### PR TITLE
Add meta tags

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: data/jobs/facebook-javascript-developer.yaml [hunk 1]
```diff
@@ -94,25 +94,30 @@ teamDistribution:
...
-  - How to handle security, logs, errors, metrics, tests (unit, functional) in a Node.js
-    application.
-  - Great oral and written communication in English.
-  nices:
-  - Experience with message brokers like RabbitMQ.
-  - Experience with Cheerio or any DOM parsing library.
-  - Experience with chrome headless, remote debugging protocol.
-  - Experience with docker.
-  - Experience with Jest, Yarn.
+  - name: responsabilities
```
Comment (rubenmoya) [inline data/jobs/facebook-javascript-developer.yaml]: why `name` and not `id` like in other places?

File: src/containers/Job/sections/TeamDistribution.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ const TeamDistribution = ({ data }) => (
 )
 
 TeamDistribution.propTypes = {
-  data: PropTypes.object
+  data: PropTypes.arrayOf(PropTypes.object)
```
Comment (rubenmoya) [inline src/containers/Job/sections/TeamDistribution.js]: <3

File: src/containers/Job/sections/Technologies.js [hunk 3]
```diff
@@ -67,7 +67,7 @@ const Technologies = ({ data }) => (
 )
 
 Technologies.propTypes = {
-  data: PropTypes.arrayOf(PropTypes.object)
+  data: PropTypes.arrayOf(PropTypes.string)
```
Comment (rubenmoya) [inline src/containers/Job/sections/Technologies.js]: <3

File: data/jobs/facebook-javascript-developer.yaml (hunk 1 above)
Comment (Kikobeats) [reply data/jobs/facebook-javascript-developer.yaml]: Because I print the value... I suppose it could be and id and add the final name into text schema

Comment (rubenmoya) [reply data/jobs/facebook-javascript-developer.yaml]: we have `id` for the name in a few places, we should choose one approach and go with it, I like `name` more, but that means we should change the other places

**Pull request merged**